pages/common/irssi.md

```diff
@@ -0,0 +1,27 @@
+# irssi
+
+> Text based IRC client since 1999. See also: https://irssi.org for full documentation.
```

Comment:
No need to mention "since 1999". That does not give any extra information about the tool. 

And I think we can remove the 2nd sentence also. We usually add a second sentence to mention additional information about the tool itself. If the users want to look for full documentation, they can look it up themselves. Mentioning the link here does not add anything extra.